iRecruit-ui/src/components/Feedback/Feedback.jsx

```diff
@@ -1,11 +1,55 @@
 import React from "react";
 import "./Feedback.css";
```

Comment:
You dont necessarily need to split CSS from the component.  Its debatable whether one is preferred over the other but just letting you know :)

Reply:
Do you mean I could have the styles incorporated with the component itself? Like style them as I create in the JSX file?